### ISSUE TITLE
DecimalValidator is unavailable in Django < 1.9

### ISSUE THREAD

**Issue by debnet** [tomchristie/django-rest-framework#3562]
```
  File "D:\VirtualEnvs\nouvelleoffre\lib\site-packages\rest_framework\serializers.py", line 431, in to_internal_value
    fields = self._writable_fields
  File "D:\VirtualEnvs\nouvelleoffre\lib\site-packages\django\utils\functional.py", line 59, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "D:\VirtualEnvs\nouvelleoffre\lib\site-packages\rest_framework\serializers.py", line 346, in _writable_fields
    field for field in self.fields.values()
  File "D:\VirtualEnvs\nouvelleoffre\lib\site-packages\rest_framework\serializers.py", line 339, in fields
    for key, value in self.get_fields().items():
  File "D:\VirtualEnvs\nouvelleoffre\lib\site-packages\rest_framework\serializers.py", line 939, in get_fields
    field_name, info, model, depth
  File "D:\VirtualEnvs\nouvelleoffre\lib\site-packages\rest_framework\serializers.py", line 1061, in build_field
    return self.build_standard_field(field_name, model_field)
  File "D:\VirtualEnvs\nouvelleoffre\lib\site-packages\rest_framework\serializers.py", line 1085, in build_standard_field
    field_kwargs = get_field_kwargs(field_name, model_field)
  File "D:\VirtualEnvs\nouvelleoffre\lib\site-packages\rest_framework\utils\field_mapping.py", line 134, in get_field_kwargs
    validator for validator in validator_kwarg
  File "D:\VirtualEnvs\nouvelleoffre\lib\site-packages\rest_framework\utils\field_mapping.py", line 135, in <listcomp>
    if not isinstance(validator, validators.DecimalValidator)
AttributeError: 'module' object has no attribute 'DecimalValidator'
```

https://github.com/tomchristie/django-rest-framework/blob/master/rest_framework/utils/field_mapping.py#L135

**Comment by xordoquy** [comment]
We may need something to ensure this will only apply to Django >= 1.9, maybe in compat.

**Comment by debnet** [comment]
Yep, DecimalValidator is only available in Django 1.9: http://django.readthedocs.org/en/latest/ref/validators.html#decimalvalidator
